src/nvim/popupmnu.c

```diff
@@ -219,7 +220,9 @@ void pum_display(pumitem_T *array, int size, int selected)
     if (curwin->w_p_rl) {
       pum_width = pum_col - pum_scrollbar + 1;
     } else {
-      pum_width = Columns - pum_col - pum_scrollbar;
+      assert(Columns - pum_col - pum_scrollbar >= INT_MIN
```

Comment:
It's fine, I understand what you mean. I still appreciate the clarification.

On March 23, 2015 5:11:18 PM EDT, "Eliseo Martínez" <notifications@github.com> wrote:
>> @@ -219,7 +220,9 @@ void pum_display(pumitem_T *array, int size, int
>selected)
>>      if (curwin->w_p_rl) {
>>        pum_width = pum_col - pum_scrollbar + 1;
>>      } else {
>> -      pum_width = Columns - pum_col - pum_scrollbar;
>> +      assert(Columns - pum_col - pum_scrollbar >= INT_MIN
>
>Please don't take me wrong. I'd like to discuss about that when you
>want. I just think it would be better doing it in a dedicated issue.
>I'll be opening one for that, if it hasn't been opened before, when I
>finish some tasks I'm currently doing.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/neovim/neovim/pull/2228/files#r26980390